src/Symfony/Component/Validator/Constraints/Type.php

```diff
@@ -35,9 +36,12 @@ class Type extends Constraint
      * @param string[]|null                            $groups
      * @param array<string,mixed>                      $options
      */
+    #[HasNamedArguments]
     public function __construct(string|array|null $type, ?string $message = null, ?array $groups = null, mixed $payload = null, array $options = [])
     {
         if (\is_array($type) && \is_string(key($type))) {
+            trigger_deprecation('symfony/validator', '7.2', 'Passing an array of options to configure the %s constraint is deprecated, use named arguments instead.', static::class);
```

Comment:
this does not deprecate passing options through the `options` named argument (kept as BC layer)

Reply:
fixed